scripts/linters/html_linter.py

```diff
@@ -290,8 +276,6 @@ def perform_all_lint_checks(self):
                     'HTML lint', False, [],
                     ['There are no HTML files to lint.'])]
 
-        # The html tags and attributes check has an additional
-        # debug mode which when enabled prints the tag_stack for each file.
```

Comment:
is there anything we are doing inside `check_html_tags_and_attributes` related to tag_stack?

Reply:
yes, we are checking if every opening tag has a closing tag, If not then raising the error